packages/admin/admin/src/index.ts

```diff
@@ -26,6 +26,7 @@ export { FieldSet, FieldSetClassKey, FieldSetProps } from "./common/FieldSet";
 export { HoverActions, HoverActionsClassKey, HoverActionsProps } from "./common/HoverActions";
 export { Loading, LoadingProps } from "./common/Loading";
 export { ToolbarActions, ToolbarActionsClassKey } from "./common/toolbar/actions/ToolbarActions";
+export { ToolbarActionButton } from "./common/toolbar/actions/ToolbarActionsButton";
```

Comment:
Do we need a changeset for this?

Reply:
i dont know, why should we need a changeset for this? Because the public api got extended?

Reply:
Yes

Reply:
so we are saying that we export more types and more components now - right?

Reply:
added changset for all packages / public api changes.